components/file_attachment_list/file_attachment_list.tsx

```diff
@@ -60,6 +63,26 @@ export default function FileAttachmentList(props: Props) {
     }
 
     const postFiles = [];
+
+    const filePreviews = sortFileInfos(props.filePreviews.slice().filter((f) => !sortedFileInfos.map((s) => s.clientId).includes(f.clientId)));
+    if (filePreviews.length !== 0) {
+        postFiles.push(
```

Comment:
could we please use `unshift` here to make sure that this always goes in at index `0`?